src/alert_rules/prometheus/prometheus_alerts.yaml

```
@@ -22,16 +22,26 @@
     "labels":
       "severity": "critical"
 
-  - "alert": "OpenSearchClusterYellow"
+  - "alert": "OpenSearchClusterYellowTemp"
     "annotations":
-      "message": "Cluster {{ $labels.cluster }} health status has been YELLOW for at least 20m. Some cluster replicas shards are not allocated."
-      "summary": "Cluster health status is YELLOW"
+      "message": "Cluster {{ $labels.cluster }} health status has been YELLOW for at least 20m. Shards are still still relocating or initializing. The cluster might be under heavy load."
```

Sorry, typo on my previous suggestion: 
```suggestion
      "message": "Cluster {{ $labels.cluster }} health status has been YELLOW for at least 20m. Shards are still relocating or initializing. The cluster might be under heavy load."
```